src/SurveyService.Models.Db/DbQuestion.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using System;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.SurveyService.Models.Db;
+
+public class DbQuestion
+{
+  public const string TableName = "Questions";
+
+  public Guid Id { get; set; }
+  public Guid? GroupId { get; set; }
+  public string Content { get; set; }
+  public DateTime? Deadline { get; set; }
+  public bool HasRealTimeResult { get; set; }
+  public bool IsAnonymous { get; set; }
+  public bool IsRevoteAvailable { get; set; }
+  public bool IsObligatory { get; set; }
+  public bool IsPrivate { get; set; }
+  public bool HasMultipleChoice { get; set; }
+  public bool HasCustomOptions { get; set; }
+  public Guid CreatedBy { get; set; }
+  public DateTime CreatedAtUTC { get; set; }
+  public Guid? ModifiedBy { get; set; }
+  public DateTime? ModifiedAtUTC { get; set; }
+  public bool IsActive { get; set; }
```

Comment:
please, move "IsActive" to line 23; Created.. and Modified.. usually are written in the end of Db model

Reply:
Done